.travis.yml

```diff
@@ -14,18 +12,22 @@ cache:
     - $GOPATH/pkg/mod
 
 go:
-  - '1.11.x'
   - '1.x'
 
-install:
-  - go build -o sharkey-server -v ./server
-  - go build -o sharkey-client -v ./client
-  - docker build -t server:latest -f Dockerfile .
-  - docker build -t client:latest -f DockerfileClientTest .
+
+before_install:
+  - mysql -e 'CREATE DATABASE sharkey_test;'
+
+install: skip
 
 script:
+  # Docker images
```

Comment:
at some point we could just put everything in `ci.sh` or something like that

Reply:
Yeah; and maybe doing docker + integration tests in a different build from go tests & binaries.